gradle/libs.versions.toml

```diff
@@ -4,20 +4,48 @@ brave-opentracing = '1.0.0'
 jaeger = '1.8.0'
 kotlin = '1.6.0'
 kotlin-coroutines = '1.6.1'
+managed-grpc = '1.45.0'
+managed-protobuf = '3.19.4'
```

Comment:
These versions manage in different project - micronaut-grpc. Need to remove them here and get from micronaut bom

Reply:
Hi @altro3 I don't know how can I reference those versions/libraries from micronaut-grpc project. Can you help me with an example? Thanks

Reply:
@n0tl3ss For example: `implementation libs.managed.grpc.protobuf`. Names for other modules you can find here: https://github.com/micronaut-projects/micronaut-grpc/blob/master/gradle/libs.versions.toml

Reply:
Something is probably missing when I make change to `testImplementation mn.grpc.protobuf` I get next error msg:
`Could not get unknown property 'grpc' for extension 'mn' of type org.gradle.accessors.dm.LibrariesForMn.`

Reply:
@n0tl3ss Yes, I think at the moment everything should be left as it is, because in any case, some of the modules are missing at the moment in the central bom file :-(.

But, I think, need to upgrade grpc to 1.46.0 and protobuf to 3.20.1